packages/xforms-engine/src/lib/reactivity/createItemCollection.ts

```diff
@@ -20,8 +18,11 @@ import type { ReactiveScope } from './scope.ts';
 import { createTextRange } from './text/createTextRange.ts';
 
 export type ItemCollectionControl = RankControl | SelectControl;
-type Item = RankItem | SelectItem;
 type DerivedItemLabel = ClientTextRange<'item-label', 'form-derived'>;
+export interface BaseItem {
```

Comment:
Start in this file

This is the simple base item for Rank and Select. 

Select's item will include `geometry` and `metadata`. The latter contains everything defined in the `choice` sheet from an XLS form, except for the value's column (usually called "name", but it can be another column), itextId, and geometry, to prevent duplicate data in Select's item.